test/phoenix/live_dashboard/system_info_test.exs

```diff
@@ -96,6 +96,57 @@ defmodule Phoenix.LiveDashboard.SystemInfoTest do
     end
   end
 
+  describe "os_mon" do
+    Application.start(:sasl)
+    Application.start(:os_mon)
```

Comment:
`Application.ensure_all_started(:os_mon)` should cover both. Let's call it in the `test_helper.exs` too. WDYT?

Reply:
ok - should the os_mon module still be required to add as an extra application ?

Reply:
It should be required to be added as an application, yes!